src/components/HomeBanner/HomeBanner.jsx

```diff
@@ -1,23 +1,76 @@
-import { mockBlogs } from "../../data/mockData";
-import { getCategoryById } from "../../utils/helperData";
+import PropTypes from "prop-types";
 import "./HomeBanner.scss";
 
-const HomeBanner = () => {
-  const authUser = JSON.parse(localStorage.getItem("authUser"));
-  const category = getCategoryById(mockBlogs[1].categoryId);
+const HomeBanner = ({ authUser, blog }) => {
   return (
-    <div className="home-banner">
-      <div className="home-banner__background-image"></div>
-      {authUser ? (
-        <h2 className="home-banner__blog-title">{mockBlogs[1].title}</h2>
-      ) : (
-        <div className="home-banner__card">
-          <h2 className="home-banner__card_blog-title">{mockBlogs[1].title}</h2>
-          <p className="home-banner__card_category-name">{category.name}</p>
+    <div
+      className="home-banner"
+      style={{ backgroundImage: `url(${blog?.bannerImage})` }}
+    >
+      <div
+        className={!authUser ? "home-banner__card" : "home-banner__auth-card"}
```

Comment:
```suggestion
        className={authUser ? "home-banner__auth-card" : "home-banner__card"}
```
always check with a truthy value